src/middlewared/middlewared/validators.py

```diff
@@ -143,6 +146,11 @@ def __call__(self, value):
             raise ValueError('Please provide a valid MAC address')
 
 
+class ReplicationSnapshotNamingSchema:
+    def __call__(self, value):
+        validate_snapshot_naming_schema(value)
```

Comment:
It feels weird to have a validator which is only going to be used for replication/snapshots here.

What do you think about having it defined in replication plugin and import it in snapshot task?

Reply:
Fair enough